docs/faq/end-users.md

```diff
@@ -1,49 +1,84 @@
 
-## Which tokens can I transact with?
+# How do I set up my Hermez Wallet? 
 
-While Hermez can support any ERC-20 token, it's primarily designed with high-frequency tokens like ETH, DAI, Tether, and wBTC in mind.
+From a desktop computer, please visit the <a href="https://hermez.io/wallet/"> Hermez Wallet </a> landing page and follow the instructions for setup. 
 
-## How long does a transaction take?
 
-While a coordinator will receive a transaction almost instantly, they need to forge the blocks and generate the zk-proofs as a first stage.
+# Transactions
 
-Since both data and zk-proofs of correctness need to be included on the Ethereum base-chain, the time for data to be reliably included (ideally finalized) on-chain depends on the Ethereum block time.
+# How long do transfers take on Hermez Network from start to finish? 
 
-To quote [Vitalik](https://vitalik.ca/general/2019/12/26/mvb.html):
+We expect that transaction time should be between 45 seconds and 15 minutes. There are several factors that can effect the transaction time, including; coordinator configuration, backlogged transactions and the transaction type. Deposits and Withdrawals take place on both L2 (Hermez) and L1 (Ethereum), which can cause an increase in transaction time depending on L1 volume.  
 
-> **The block time of the base layer sets the latency for anything whose confirmation depends things being included in the base layer.** This could be worked around with on-chain security deposits, aka "bonds", at the cost of high capital inefficiency, but such an approach is inherently imperfect because a malicious actor could trick an unlimited number of different people by sacrificing one deposit.
 
-We expect that some systems can be developed as a new layer on top of zk-rollups where coordinators can guarantee finality of transactions with instant confirmations (a sort of economic finality for low-value transactions).
+# Who can I transact with on Hermez Network? 
 
-## How are transaction fees paid by users?
+For transfers on Hermez, both the sender and receiver must be on L2. However, the sender may transfer funds to a receiver who has not yet created account on Hermez L2 as long as the receiver has opened the Hermez Wallet app with their Metamask account.  
 
-The default is to pay fees in the same token you’re sending,
 
-But with two atomic (linked) transactions you can pay a fee in a different token.
+# When I send my funds to Hermez on Layer 2, how secure is it?  
 
-## What if the ERC-20 token I wish to transact with is not yet supported?
+It is as secure as it would be on the Layer 1, Ethereum blockchain. Hermez has completed 3 security audits before launching the mainnet to ensure network security.   
+For more information on Hermez security please visit, https://docs.hermez.io/#/about/security
 
-Most common tokens will be preloaded in the system. In any case, there is a rollup function for this. But you'll need to pay a small fee in order to add a token.
 
```

Comment:
Add new question ### Can coordinators take my money?
No (we probably need to expand a bit)

New question ### Can coordinators censor transactions?

Coordinators can decide which transactions they select to include in the batch to forge.

In case that a coordinator censures a transaction, users have the option to force coordinators to include their transaction.